src/components/PlaybackControls/index.tsx

```diff
@@ -33,9 +35,15 @@ const PlayBackControls = ({
     onTimeChange,
     loading,
     timeStep,
+    timeUnits,
     isEmpty,
 }: PlayBackProps) => {
-    const [unitIndex, setUnitIndex] = useState(0);
+    const [unitLabel, setUnitLabel] = useState("s");
+    useEffect(() => {
```

Comment:
I'm not sure why you're setting this in local state here since it's coming from app state. Couldn't you just do `timeUnits.name || "s"` when you're setting the label down below?

Reply:
I was wondering the same thing

Reply:
You're totally right, I removed the local state and now just rendering `timeUnits ? timeUnits.name : "s"`